proposal/color-4-hwb.md

```diff
@@ -152,6 +216,84 @@ adjust($color,
   $alpha: null)
 ```
 
+This function's new definition is as follows:
+
+* If `$color` isn't a color, throw an error.
+
+* Let `alpha` be `$color`'s alpha channel.
+
+* If `$alpha` isn't null:
+
+  * If `$alpha` isn't a number between -1 and 1 (inclusive), throw an error.
+
+  * Set `alpha` to `alpha + $alpha` clamped between 0 and 255.
+
+* If `$hue` isn't a number or null, throw an error.
+
+* If any of `$red`, `$green`, or `$blue` aren't null:
+
+  * If any of `$hue`, `$saturation`, `$lightness`, `$whiteness`, or `$blackness`
+    aren't null, throw an error.
+
+  * If any of `$red`, `$green`, or `$blue` aren't either null or numbers between
+    -255 and 255 (inclusive), throw an error.
+
+  * Let `red`, `green`, and `blue` be `$color`'s red, green, and blue channels.
+
+  * If `$red` isn't null, set `red` to `red + $red` clamped between 0 and 255.
+
+  * If `$green` isn't null, set `green` to `green + $green` clamped between 0 and 255.
+
+  * If `$blue` isn't null, set `blue` to `blue + $blue` clamped between 0 and 255.
+
+  * Return a color with `red`, `green`, `blue`, and `alpha` as the red, green,
+    blue, and alpha channels, respectively.
+
+* Otherwise, if either `$saturation` or `$lightness` aren't null:
+
+  * If either `$whiteness` or `$blackness` aren't null, throw an error.
+
+  * If either `$saturation` or `$lightness` aren't either null or numbers
+    between -100 and 100 (inclusive), throw an error.
```

Comment:
Here and below (lines 265, 268, 347): should these have unit `%`?

Reply:
Unfortunately, this represents the current behavior. We currently ignore units for saturation and lightness no matter what, which isn't ideal but isn't necessarily enough of a problem to warrant a deprecation pass either.